invadedhouse/commands.py

```diff
@@ -18,7 +18,8 @@ def run(self):
             cmd = self.get_next_command()
             self.interpret_command(cmd)
 
-    def get_next_command(self) -> str:
+    @staticmethod
```

Comment:
this is called an annotation. it tells the python interpreter that this function can be called without an instance of CommandInterpreter... you'll eventually learn about the difference between static, class, and instance methods but [I'll leave a link here](https://stackabuse.com/pythons-classmethod-and-staticmethod-explained/) if you're interested in learning more now.